contribs/gmf/less/desktop.less

```diff
@@ -20,6 +20,10 @@
 @toolbar-width: 4.5rem;
 @border-color: darken(@brand-primary, @standard-variation);
 @search-width: 8 * @map-tools-size;
+@font-size-base: 13px;
+@padding-base-vertical: 5px;
+@padding-base-horizontal: 10px;
+@form-group-margin-bottom: 10px;
```

Comment:
In bootstrap 4, they are still using a pixel value for the font-size of the html element. But they will introduce a new `font-size-root` which doesn't exist yet.
https://github.com/twbs/bootstrap/blob/v4-dev/scss/_variables.scss#L166-L172

Also variables for the padding have completely changed in v4. They will called "spacers". The current code won't be relevant in v4 so I'm not really sure we should pay to much attention to it right now.